Content.Shared/Clothing/EntitySystems/MaskSystem.cs

```diff
@@ -77,6 +78,7 @@ private void ToggleMaskComponents(EntityUid uid, MaskComponent mask, EntityUid w
     private void OnFolded(Entity<MaskComponent> ent, ref FoldedEvent args)
     {
         ent.Comp.IsToggled = args.IsFolded;
+        Dirty(ent, ent.Comp);
 
         ToggleMaskComponents(ent.Owner, ent.Comp, ent.Owner);
```

Comment:
Wouldn't it be better to just do a dirty call in togglemaskcomponents as it covers all 3 code paths